numpy/lib/index_tricks.py

```diff
@@ -68,6 +71,15 @@ def ix_(*args):
     array([[2, 4],
            [7, 9]])
 
+    >>> ixgrid = np.ix_([True,True], [2,4])
+    >>> a[ixgrid]
+    array([[2, 4],
+           [7, 9]])
+    >>> ixgrid = np.ix_([True,True], [False, False, True, False, True])
```

Comment:
Which one should be preffered? No spacing after `,` in the lists or whitespaces after each `,`? 

`[False,False,True,False,True]` looks a bit cramped.